webhook/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "pelias-update-webhook",
+  "version": "1.0.0",
+  "main": "index.js",
+  "author": "IBI Group",
+  "license": "MIT",
+  "scripts": {
+    "start": "node index.js"
+  },
```

Comment:
Let's at least add linting via tsdx in here. Also, there should be some GitHub Actions to run the linting process.

Reply:
Good call. Added in dbab857d611c1d6aaf18033fba41217c69a193e2